magpie/ui/network/__init__.py

```diff
@@ -0,0 +1,11 @@
+from magpie.utils import get_logger
+
+LOGGER = get_logger(__name__)
+
+
+def includeme(config):
+    from magpie.ui.network.views import NetworkViews
+    LOGGER.info("Adding UI network...")
+    path = "/ui/network/authorize"
+    config.add_route(NetworkViews.authorize.__name__, path)
```

Comment:
I'm confused...

This is a UI endpoint, not API

Reply:
You're right. I got mixed by the `add_route` using `NetworkViews` and incorrectly assumed it was a CorniceSwagger service. 
Just in case, can you adjust `NetworkViews.authorize.__name__` by `fully_qualified_name(NetworkViews.authorize)` so that it does not conflict with other "authorize" if more are added later?